docs/cimpl6Authoring.md

```diff
@@ -39,56 +37,56 @@ Model key:
+The user must however invest time to understand the OBF logical model. Also, OBF does not comprehensively support all FHIR **resource**s, especially the ones introduced in FHIR R4 with a low maturity level.
 
 ## Logical Model
 
-Start by understanding the model you want to document in CIMPL, and its relationship to either base FHIR resources and profiles, or to OBF.
+Start by understanding the model you want to document in CIMPL, and its relationship to either base FHIR **resource**s and **profile**s, to OBF, or to a combination.
 
-Keeping in mind that CIMPL is primarily a way to create logical models with the capability to _model-once, translate-to-many_, the modeling author should consider requirements-gathering and high level modeling steps.  While one approach is proposed below, the modeling author is not limited to following these steps and might find better approaches to creating detailed clinical models.
+Keeping in mind that CIMPL is primarily a way to create logical models with the capability to _model-once, translate-to-many_, the modeling author should consider requirements gathering and high level modeling steps.  While one approach is proposed below, the modeling author is not limited to following these steps and might find better approaches to creating detailed clinical models.
 
 * Define the use cases behind the creation of a model.
-* Create a high-level conceptual model which addresses your defined use case and can be easily understood by both technical and clinical communities.
-* Create a data dictionary listing the data elements, cardinality, and potential value sets involved if the data type is a coded element. This provides a convenient summary for implementers presented in a way that can be understood by non-technical subject matter experts involved in defining the use cases.
+* Create a high-level conceptual model to address your use case that can be easily understood by both technical and clinical communities.
+* Create a data dictionary listing the data elements, cardinality, and potential value sets involved if the data type is a coded element. This provides a convenient summary for implementers presented in a way that can be understood by non-technical subject matter experts.
 * Start documenting your model with CIMPL.
 
 ## Model Folder Structure and Source Control
 
-Create a folder/directory to contain all files you create for the model. If you are modeling with a distributed team, you should consider using a source control system for you CIMPL files.
+Create a folder/directory to contain all files you create for the model. If you are modeling with a distributed team, you should consider using a source control system for your CIMPL files.
 
-[CIMPL In-Depth Tutorial](#cimpl6Tutorial_detail.md)
+[CIMPL Tooling Reference Manual](cimpl6ToolingReference.md#suggested-directory-structure)
```

Comment:
Instead of the text saying "CIMPL Tooling Reference Manual", suggest changing to "Suggested Directory Structure for Your Project" or something along those lines. Otherwise it looks like you are linking to a the entire document, not a selected section.

Reply:
Done

Reply:
Please change the text of the link.